tests/test_runner/test_parallel.py

```diff
@@ -50,6 +50,15 @@ def dummy_test(self):
             with self.subTest(index=i):
                 self.assertEqual(i, 1)
 
+    def pickle_error_test(self):
+        """A dummy test for testing TestCaseDTO."""
+        # ^ note: 3.6's implementation of TestCase.shortDescription()
+        #   doesn't strip whitespace before taking the first line of the
+        #   docstring, so the single-line docstring is part of this test
```

Comment:
The current development version of Django support Python 3.8+, so this note and workaround are unnecessary.